src/components/Upload.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <UiLoading v-if="loading" />
+  <label v-else class="file-select">
+    <input type="file" @change="handleFileChange" accept="image/*, video/mp4" />
```

Comment:
We should accept only image here
```suggestion
    <input type="file" @change="handleFileChange" accept="image/*" />
```